docs/modules/ROOT/pages/java.adoc

```diff
@@ -462,7 +465,16 @@ documents whose `public_likes` array property contain a value equal to
 
 [source,java]
 ----
-include::{examplesdir}/QueryAPITest.java[tags=query-collection-operator]
+include::{examplesdir}/QueryAPITest.java[tags=query-collection-operator-contains]
+----
+
+===== IN Operator
+
+The `IN` operator is useful when you need to explicitly list out the values to test against. The following example looks for documents whose `first`, `last` or `username` property value equals "Armani".
+
+[source,java]
+----
+include::{examplesdir}/QueryAPITest.java[tags=query-collection-operator-in]
```

Comment:
@hideki could you add this example in `QueryAPIText.java`? Swift model https://github.com/couchbase/couchbase-lite-ios/pull/2153/commits/4fa643df554ea88b510bfb5f3f7ecbbccfb3e589#diff-4c45034a19156ebaf70458b839dd74e8

Reply:
done